examples/bpk-component-swap-button/examples.js

```diff
@@ -0,0 +1,30 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/* @flow strict */
+
+import BpkSwapButton from '../../packages/bpk-component-swap-button';
+import { action } from '../bpk-storybook-utils';
+
+const DefaultExample = () => (
+
+  <div style={{ position: 'absolute', top:'20%', left:'20%' }}>
+  <BpkSwapButton label="Backpack" ariaLabel='Swap Button' ariaLiveTextProp='Swapped' onClick={action('Button clicked')}/>
```

Comment:
I would remove this label prop as its not part of the props on the component so may cause confusion

Reply:
leftover from boilerplate template, thanks for spotting